docs/validators/security.md

```diff
@@ -39,3 +39,14 @@ Sentry Nodes should edit their config.toml:
 # Comma separated list of peer IDs to keep private (will not be gossiped to other peers)
 private_peer_ids = "ipaddress of validator nodes"
 ```
+
+## Environment Variables
```

Comment:
Where did this come from?

Reply:
Ohhh, Looks like this address #1973. Wanna tag that somewhere (at least in the PR body)